packages/manager/modules/veeam-enterprise/src/dashboard/license/license.controller.js

```diff
@@ -10,11 +10,19 @@ export default class VeeamEnterpriseLicenseCtrl {
   }
 
   dismissModal() {
+    this.trackClick(
+      `license::${this.action === 'register' ? 'activate' : 'update'}::cancel`,
+    );
     this.goToDashboard();
   }
 
   submitForm(form) {
     if (form.$valid) {
+      this.trackClick(
+        `license::${
+          this.action === 'register' ? 'activate' : 'update'
```

Comment:
Nitpick: reuse register and activate strings by extracting them into constants.

Reply:
I have delegated in a function as suggested in https://github.com/ovh/manager/pull/6765#discussion_r841451303